api/models/article.coffee

```diff
@@ -38,6 +38,9 @@ module.exports = class Article extends Backbone.Model
   isFeatured: ->
     @get('featured')
 
+  hasInstantArticle: ->
```

Comment:
👍  I still find this a clever use of Backbone to add model logic on the server-side. However, I do wonder if it'd be simpler to do a "model library" where we export "pure" functions in this case, e.g.

````javascript
export const hasInstantArticle = (article) => {
  return article.featured && _.contains(["standard", "feature"], article.layout) 
}
````

This more vanilla/functional approach, for instance, feels more natural to the Redux-ey code we're moving towards on the client-side. Just food for though!